scc/src/main.rs

```diff
@@ -49,18 +60,46 @@ fn main() -> Result<(), Error> {
     Ok(())
 }
 
-fn setup_logger(cache_dir: &Path) -> Result<(), Error> {
+pub fn exists(file : &Path) -> bool {
+    fs::metadata(file).is_ok()
+}
+
+fn setup_logger(cache_dir: &Path, date_time_logger : bool) -> Result<(), Error> 
```

Comment:
I think a more appopriate name would be `include_date_in_filename: bool` or something like that
```suggestion
fn setup_logger(cache_dir: &Path, include_date_in_filename: bool) -> Result<(), Error> 
```